ru/lessons/ecto/associations.md

```diff
@@ -0,0 +1,500 @@
+---
+version: 1.1.0
+title: Ассоциации
+---
+
+В этом уроке мы рассмотрим как использовать Ecto для определения и работы с ассоциациями между структурами (schema).
+
+{% include toc.html %}
+
+## Настройка
+
+Мы начнем с приложения `Example`, которое уже использовалось в предыдущих уроках. [Здесь](../basics) можно вспомнить о чём идёт речь.
+
+## Типы ассоциаций
+
+Существует три типа связей которые мы можем обьявить между структурами в Ecto. Мы рассмотрим какие они есть и как работать с каждым из них.
+
+### Belongs To/Has Many
+
+Мы добавим несколько новых сущностей в приложение для того чтобы пользователи могли начать каталогизировать свои любимые фильмы. Мы начнем с добавления двух структур: `Movie` и `Character`. Мы добавим двухстороннуюю связь между ними: у фильма есть много персонажей и каждый персонаж пренадлежит к какому-то одному фильму.
```

Comment:
```suggestion
Добавим несколько новых сущностей в приложение, чтобы пользователи могли начать каталогизировать свои любимые фильмы. Начнём с добавления двух структур: `Movie` и `Character`. Добавим двухстороннуюю связь между ними: у фильма есть много персонажей, и каждый персонаж принадлежит какому-то одному фильму.
```

Probably it's also worth to leave references to the original terms `belongs to/has many` in this paragraph.